### ISSUE TITLE
resetDataCustodianDB management command allows DB to be reset using client_access_token, upload_access_token and registration_access_token

### ISSUE THREAD

**Issue by dfcoffin** [energyos/OpenESPI-DataCustodian-java#217]
Only the datacustodian_access_token should allow the Sandbox Data Custodian DB to be reset {or initialized}

**Comment by dfcoffin** [comment]
Closed 49b2c1ee92b9de65d59e223915af03b1a2721a92

**Comment by MartyBurns** [comment]
Why did you reverse the requirement? Did you run regression test against change?

**Comment by dfcoffin** [comment]
I didn't reverse the requirement.  We said only the data_custodian_access_token can be used to issue a /management API command when we reviewed the new selection on the API Documentation page.  At the time, the only tokens which didn't allow the /management API request to be processed were Alan's and Charles' access_tokens.

What I discovered was the ResourceValidationFilter was testing for /resource in the path statement and if not present, was failing to verify the presence of an Authorization header.  The next test was to determine if the request contained an Authorization header or was a /resource API request.  Therefore, all of the /management API requests were immediately exiting the ResourceValidationFilter without confirming the request was using the data_custodian_access_token.

The change I made was to ensure all API request execute the /resource API test and the Authorization header presence test, so the logic that was inserted to verify only /management API request containing data_custodian_access_tokens were allowed to execute /management API requests.
